### PR TITLE
MRG allow passing axes to evoked.plot_joint

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
evoked.plot_joint is a figure-level function. There is no easy way around this. But currently, we make it unnecessarily hard to embed if with other figures, by checking for axes keywords for the dict args. This removes that check, and allows manually forwarding axes. Thus, composite images are possible.
However, this is rather complicated to use ...

Not sure about it to be honest. I use it locally, but it might not be useful for others due to the complexity.

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1252,11 +1252,20 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
...
                 ", ".join(list(illegal_args))))
+    if len(ts_args.get("axes", "1")) != 1:
+        raise ValueError("If `ts_args` contains 'axes', it must contain "
+                         "exaclty one element.")
+    n_topomaps = (3 if times is None else len(times)) + 1
+    if "axes" in topomap_args:
+        if n_topomaps != len(topomap_args["axes"])
+            raise ValueError("If `topomap_args` contains 'axes', it "
+                             "must contain one more element than the number "
+                             "of times you want to plot.")
```
Comment (larsoner) [inline mne/viz/evoked.py]: be nice to people, give them this desired value and the value they gave :)

**Comment by jona-sassenhagen** [comment]
Actually this one is the same stuck Travis job.

**Comment by agramfort** [comment]
can you show an image you managed to plot nicely with this?

**Comment by agramfort** [comment]
ok can you just add a what's new entry?

**Comment by larsoner** [comment]
Thanks @jona-sassenhagen

**Comment by jona-sassenhagen** [comment]
Hu, thanks Eric.

**Pull request merged**